server.Dockerfile

```diff
@@ -1,28 +1,25 @@
-FROM golang:1.6.0
+FROM golang:1.6.0-alpine
 MAINTAINER David Lawrence "david.lawrence@docker.com"
 
-RUN apt-get update && apt-get install -y \
-    libltdl-dev \
-    --no-install-recommends \
-    && rm -rf /var/lib/apt/lists/*
-
-EXPOSE 4443
+RUN apk add --update git gcc libc-dev && rm -rf /var/cache/apk/*
 
 # Install DB migration tool
-RUN go get github.com/mattes/migrate
+RUN go get -ldflags "-extldflags -fno-PIC" github.com/mattes/migrate
```

Comment:
Because of https://github.com/golang/go/issues/14851#issuecomment-200915770 - without that the `migrate` binary segfaults.  :|  If the golang-alpine image gets upgraded, we can probably remove this.